docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+# Deploying to AWS
+
+!!! danger "AWS Admin Privileges Required"
+
+    This deployment can only be run by AWS administrators, so we assume the user has a basic understanding of code
+    development, Docker containers, and the AWS cloud.
+
+## Overview
+
+The goal of this guide is to help you create a functioning pipeline in AWS that allows you to process data on-the-fly as
+it enters S3 buckets (Ingest applications), or on a regular cron schedule (VAP applications), and store the processed
+results in an output S3 bucket in a structured format.
```

Comment:
Add this:

```Your Tsdat pipelines will be deployed via an AWS Cloud Formation stack. The stack creates an AWS CodeBuild project that is connected to your GitHub repository so that an AWS deployment is triggered every time you commit code changes.  The following images illustrate different aspects of the deployment.  The first image shows how the deployed pipelines will function on AWS; the second image shows all the specific AWS resources that are included in the stack, and the final image shows how the CodeBuild build/deploy cycle works.```

Reply:
Adding this